apps/console/src/features/organizations/components/edit-organization-role/organization-role-permission.tsx

```diff
@@ -0,0 +1,291 @@
+/**
+ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.com) All Rights Reserved.
```

Comment:
Change license header. Check in other places as well

Reply:
Done in https://github.com/wso2/identity-apps/pull/3617/commits/630fa8dda5d13075f2d424b2493249572e255217